modules/reactor/test/tests/meshgenerators/peripheral_ring_mesh_generator/ring_quadratic.i

```diff
@@ -0,0 +1,33 @@
+[Mesh]
+  [pccmg]
+      type = PolygonConcentricCircleMeshGenerator
+      num_sides = 8
+      num_sectors_per_side = '4 4 4 4 4 4 4 4'
+      background_intervals = 2
+      polygon_size = 5.0
+
+      preserve_volumes = on
+
+      tri_element_type = TRI6
```

Comment:
If I  comment these two out to make the mesh linear, I get a different volumes for the external ring

Reply:
I will look into this, thanks

Reply:
Interesting, paraview says the areas are the same...(706.858 total which is the same as the linear elements values reported by MOOSE)

Maybe we need to take a look at the source code that calculates these printed volume values...

Reply:
```
[Mesh]
  [dir1]
    type = ElementGenerator
    nodal_positions = '0 0 0
                       1 0 0
                       1 1 0
                       0 1 0
                       0.5 -0.5 0
                       1 0.5 0
                       0.5 1 0
                       0 0.5 0
                       0.5 0.5 0'
    element_connectivity = '0 1 2 3 4 5 6 7 8'
    elem_type = 'QUAD9'
  []
[]
```
The element area is 1.25 but MOOSE says 1.333 (or quadratic element area has a different definition?)

Reply:
@roystgnr thoughts?

Reply:
I think when you say 1.25 you have a triangle between the three slopes at the bottom
shouldnt it be quadratic?

Reply:
I just tried to give an extreme case. There might be some rules that I am not aware of. Let's see if @roystgnr has any insights.

Reply:
> I think when you say 1.25 you have a triangle between the three nodes at the bottom of the element shouldnt it be quadratic?

Ah, so the edge should be parabolic for QUAD9 element. In that case the radius correction function needs to be modified.

Reply:
I think so.
Irrelevant here but I think it might be the same for quad8?

Reply:
fixed. sorry for the batch regolding...

Reply:
> Ah, so the edge should be parabolic for QUAD9 element

This is correct.  In the `ElementGenerator` case in particular, 4/3 is the correct area.